control-plane-operator/hostedclusterconfigoperator/controllers/resources/resources.go

```diff
@@ -2519,8 +2519,15 @@ func (r *reconciler) reconcileStorage(ctx context.Context, hcp *hyperv1.HostedCo
 		errs = append(errs, fmt.Errorf("failed to reconcile Storage : %w", err))
 	}
 
-	if hcp.Spec.Platform.Type == hyperv1.AWSPlatform {
-		driver := manifests.ClusterCSIDriver(operatorv1.AWSEBSCSIDriver)
+	var driverNames []operatorv1.CSIDriverName
+	switch hcp.Spec.Platform.Type {
+	case hyperv1.AWSPlatform:
+		driverNames = []operatorv1.CSIDriverName{operatorv1.AWSEBSCSIDriver}
+	case hyperv1.OpenStackPlatform:
+		driverNames = []operatorv1.CSIDriverName{operatorv1.ManilaCSIDriver}
```

Comment:
I'll conflict with Cinder I guess, but we can rebase.

Reply:
I was polite enough [to call out the need to add Manila in the Cinder change](https://github.com/openshift/hypershift/pull/4936/commits/b8a63f1c1d3a6e77e627cc04ec9d8e9f683879e3#diff-c888020c5dc46c458d818b931db97131d6e35b90661fe1030a39ebeac8859b19R2527) :smiling_face_with_tear: (kidding :wink:)